assembl/static2/js/app/components/debate/thread/announcement.jsx

```diff
@@ -62,7 +64,7 @@ class Announcement extends React.Component {
     const { ideaWithPostsData: { idea }, announcementContent } = this.props;
     const { numContributors, numPosts, posts } = idea;
     const sentimentsCount = getSentimentsCount(posts);
-    const videoContent = dirtySplitHack(announcementContent);
+    const mediaContent = dirtySplitHack(announcementContent);
```

Comment:
You should ask to a backend developer if it's complicated to adapt the announcement to avoid this hack

Reply:
We need this feature for the phase 1. So we have what we need in the admin section.

Reply:
There is the admin interface in the phase 1